src/realm-settings/KeysProvidersTab.tsx

```diff
@@ -93,7 +95,7 @@ export const KeysTabInner = ({ components, refresh }: KeysTabInnerProps) => {
   const [liveText, setLiveText] = useState("");
 
   useEffect(() => {
-    setItemOrder(["data", ...itemIds]);
+    setItemOrder([...itemIds]);
```

Comment:
If the value is already an array there should be no need to spread it.
```suggestion
    setItemOrder(itemIds);
```